.github/workflows/graphql.yml

```diff
@@ -0,0 +1,26 @@
+name: GraphQL API Documentation
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2

```

Comment:
This should checkout appropriate branch with right access tokens. For e.g look https://github.com/meshery/meshery/blob/master/.github/workflows/error-codes-updater.yaml